src/bors/merge_queue.rs

```diff
@@ -83,10 +84,35 @@ pub async fn handle_merge_queue(ctx: Arc<BorsContext>) -> anyhow::Result<()> {
                                     .await?;
                             }
                             Err(error) => {
-                                tracing::error!("Failed to push to base branch: {:?}", error);
-
-                                let comment = auto_build_push_failed_comment(&error.to_string());
-                                repo.client.post_comment(pr.number, comment).await?;
+                                let current_base_sha =
+                                    repo.client.get_branch_sha(&pr.base_branch).await?;
+                                let build_time_base_sha = CommitSha(auto_build.parent.clone());
+
+                                if current_base_sha != build_time_base_sha {
+                                    tracing::info!(
+                                        "Base branch has moved for PR {pr_num} (was: {}, now: {}). Deleting auto build.",
+                                        build_time_base_sha,
+                                        current_base_sha
+                                    );
+
+                                    let comment = auto_build_base_moved_comment(
```

Comment:
Is this something that homu handles? I never saw this occur in practice :laughing: Can we perhaps examine the returned `error` to check what exactly has failed on the GitHub side?